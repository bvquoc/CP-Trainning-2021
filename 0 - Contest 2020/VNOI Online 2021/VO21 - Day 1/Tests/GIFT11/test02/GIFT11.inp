10
8
87 3645 1 455067963 786667 2 7859829 34
8
40551227 8 564176 9648 230278429 8012 82383 8
8
95 4322 573 1645014 52363277 663984349 608157554 5017506
8
44 664 210808799 382427943 1 7554658 1669813 36767
8
39536 303334 23 2809 207 888149299 707281 17926543
8
30 96060 240785281 771999765 821992 4 79470 961339
8
1 4082714 415693 7578 64301732 736 896 50133045
8
958110 5 947641591 65474378 71 7433212 674 15
8
81 12972 53860133 151192595 5368 8 31420410 377070
8
2448683 197 615 5243655 8862688 3240 532970 241
