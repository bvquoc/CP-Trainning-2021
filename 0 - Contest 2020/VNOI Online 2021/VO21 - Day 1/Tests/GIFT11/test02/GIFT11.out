5184
1728
3744
3456
3840
5184
4032
3264
5376
11520
